modules/exploits/linux/http/hp_van_sdn_controller.rb

```diff
@@ -0,0 +1,72 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'HP VAN SDN Controller Hardcoded Service Token RCE',
+      'Description'    => %q{
+        This module exploits a hardcoded service token and weak sudo permissions
+        in HP(E) VAN SDN Controller <= 2.7.18.0503 to execute commands as root.
+      },
+      'Author'         => [
+        'Matt Bergin', # Vulnerability discovery
```

Comment:
and exploit

Reply:
I assume Matt wrote the PoC but wasn't sure if it was a combined KoreLogic effort. I'll add it to Matt's name.